spec/requests/review_spec.rb

```diff
@@ -2,6 +2,17 @@
 
 require 'rails_helper'
 
+RSpec.shared_examples 'invalid score' do |klass|
```

Comment:
unused argument?
`_unsused` `_klass` in its front is helpful when reading the code

Reply:
btw can this be removed?

Reply:
removed